css/style.css

```diff
@@ -0,0 +1,1137 @@
+}
+
+.main__container {
+  display: flex;
+  margin-top: 510px;
+  padding: 0 250px;
+  max-width: 100%;
+  align-items: center;
+  text-align: center;
+  justify-content: space-around;
+}
+
+.main__container-first {
+  display: flex;
+  justify-content: space-between;
+}
+
+.main__container-second {
+  display: flex;
+  justify-content: space-between;
+  position: relative;
+  bottom: 14px;
+  margin-right: 33px;
+}
+
+.main__row {
+  display: flex;
+  flex-wrap: wrap;
+  align-items: baseline;
+  justify-content: space-between;
+  margin: 0 62px 0 26px;
+}
+
+.main__img-vector {
+  float: left;
+  padding-left: 10px;
+}
+
+.main__img-stroke {
+  float: left;
+  width: 41.83px;
+  margin-right: -10px;
+  /* position: relative; */
+}
+
+.main__row-title {
+  display: flex;
+  align-items: baseline;
+  padding-left: 15px;
+  margin-left: -15px;
+  font-style: bold;
+  font-weight: 600;
+  font-size: 35px;
+  color: black;
+  position: relative;
+  right: 3px;
+  margin-top: -10px;
+  font-family: "Montserrat", sans-serif;
+  line-height: 43px;
+}
+
+.main__subtitle-time {
+  display: flex;
+  margin-left: 6px;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 20px;
+  color: black;
+  line-height: 24px;
+}
+
+.main__subtitle-dist {
+  display: flex;
+  margin-left: 6px;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 20px;
+  color: black;
+  line-height: 24px;
+}
+
+.main__row-desc {
+  display: flex;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 12px;
+  line-height: 15px;
+  color: black;
+  position: relative;
+  right: -10px;
+  margin-top: 0px;
+  margin-left: 15px;
+}
+
+.main__row-test {
+  display: flex;
+  margin-bottom: 10px;
+  align-items: center;
+  flex-direction: row-reverse;
+  text-shadow: 0 4px 4px rgba(0, 0, 0, 0.25);
+}
+
+.row__test {
+  display: flex;
+}
+
+.main__img-steering {
+  float: left;
+  margin-top: 20px;
+  margin-right: 8px;
+}
+
+.main__img-steering-1, .main__img-steering-2 {
+  display: none;
+}
+
+.main__row-text {
+  display: flex;
+  flex-wrap: row nowrap;
+  align-items: center;
+  font-weight: bold;
+  text-align: center;
+  font-size: 13px;
+  color: black;
+  margin-top: 23px;
+}
+
+.button {
+  margin: 0;
+  display: flex;
+  align-items: center;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 13px;
+  text-align: center;
+  border: 2px solid #e51937;
+  width: 160px;
+  height: 40px;
+  padding: 10px 20px;
+}
+
+.main__button {
+  position: relative;
+  color: #e51937;
+  background-color: white;
+  text-align: center;
+  justify-content: center;
+  margin-left: 52px;
+  margin-top: 11px;
+  right: 10px;
+}
+
+.main__button:hover {
+  box-shadow: 0 5px 8px #3c354a;
+  transition: 0.3s all ease-out;
+  cursor: pointer;
+}
+
+.main__arrow {
+  display: flex;
+  justify-content: center;
+  margin-top: 25px;
+}
+
+@media (max-width: 1270px) {
+  .header {
+    padding: 24px 33px 0 15.32px;
+  }
+
+  .header__img-burger {
+    width: 35.23px;
+  }
+  
```

Comment:
Codacy found an issue: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/Tesla-Project_Evgen/pullRequest?prid=6071575)